teachers_digital_platform/tests/models/test_activity_page.py

```diff
@@ -0,0 +1,91 @@
+from django.test import TestCase
+
+from wagtail.wagtaildocs.models import Document
+
+from model_mommy import mommy
+from teachers_digital_platform.models import (
+    ActivityAgeRange, ActivityBloomsTaxonomyLevel, ActivityBuildingBlock,
+    ActivityCouncilForEconEd, ActivityDuration, ActivityGradeLevel,
+    ActivityJumpStartCoalition, ActivityPage, ActivitySchoolSubject,
+    ActivityTeachingStrategy, ActivityTopic, ActivityType
+)
+
+
+class TestActivityPage(TestCase):
+    fixtures = ['tdp_initial_data']
+
+    def setUp(self):
+        super(TestActivityPage, self).setUp()
+
+    def test_get_subtopic_ids_returns_correct_subtopics(self):
+        # Arrange
+        activity_page = self.create_activity_detail_page('Test', 'test')
+        # Act
+        actual_subtopic_ids = activity_page.get_subtopic_ids()
+        # Assert
+        self.assertTrue(6 not in actual_subtopic_ids)
+        self.assertTrue(7 in actual_subtopic_ids)
+
+    def test_get_subtopic_ids_works_with_no_topics(self):
+        # Arrange
+        activity_page = self.create_activity_detail_page('Test', 'test', topic_list=[])  # noqa: E501
+        # Act
+        actual_subtopic_ids = activity_page.get_subtopic_ids()
+        # Assert
+        self.assertTrue(isinstance(actual_subtopic_ids, (list, set)))
+        self.assertFalse(actual_subtopic_ids)
+
+    def test_get_subtopic_ids_works_with_no_subtopics(self):
+        # Arrange
+        activity_page = self.create_activity_detail_page('Test', 'test', topic_list=[6])  # noqa: E501
+        # Act
+        actual_subtopic_ids = activity_page.get_subtopic_ids()
+        # Assert
+        self.assertTrue(isinstance(actual_subtopic_ids, (list, set)))
+        self.assertFalse(actual_subtopic_ids)
+
+    def test_get_grade_level_ids_returns_correct_grade_levels(self):
+        # Arrange
+        activity_page = self.create_activity_detail_page('Test', 'test')
+        # Act
+        actual_grade_level_ids = activity_page.get_grade_level_ids()
+        # Assert
+        self.assertTrue(2 in actual_grade_level_ids)
+        self.assertTrue(1 == len(actual_grade_level_ids))
```

Comment:
```suggestion
        self.assertEqual(len(actual_grade_level_ids), 1)
```

I _think_ it's preferable to put the thing you're checking first, followed by the value you're expecting.